<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ingoalsWindow</class>
 <widget class="QMainWindow" name="ingoalsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SavingsButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新增储蓄计划</string>
    </property>
   </widget>
   <widget class="QListView" name="accountListView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>71</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="savingtableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>631</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="category_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="completionprogressBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="category_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="category_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="completion_progress">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>完成进度</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_progress">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时间进度</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="timeprogressBar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="advice_1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="advice_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>621</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="advice_3">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>480</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupersonal_financial_manager">
    <property name="title">
     <string>personal financial manager 财务目标 储蓄计划</string>
    </property>
   </widget>
   <addaction name="menupersonal_financial_manager"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
